types/franc/franc-tests.ts

```diff
@@ -0,0 +1,14 @@
+import * as detect from 'franc';
```

Comment:
Since `detect` is a function, it should be `import detect = require('franc')` here: https://github.com/DefinitelyTyped/DefinitelyTyped#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports

Reply:
I just fixed this…

@plantain-00: If there are any more changes you'd like to suggest, could you please do them all in one go as opposed to having to do this constant back & forth? Thanks!